test/e2e/lib/resources/kube.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/golang/protobuf/proto"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	pkgTest "knative.dev/pkg/test"
+)
+
+func AuditLogsTargetJob(name string, envVars []v1.EnvVar) *batchv1.Job {
+	return baseJob(name, "auditlogs_target", envVars)
+}
+
+func StorageTargetJob(name string, envVars []v1.EnvVar) *batchv1.Job {
+	return baseJob(name, "storage_target", envVars)
+}
+
+func TargetJob(name string, envVars []v1.EnvVar) *batchv1.Job {
+	return baseJob(name, "target", envVars)
+}
+
+func SenderJob(name string, envVars []v1.EnvVar) *batchv1.Job {
+	return baseJob(name, "sender", envVars)
+}
+
+func baseJob(name, imageName string, envVars []v1.EnvVar) *batchv1.Job {
```

Comment:
maybe add a comment as all of the above use them?

Reply:
Added the comment.